modules/electrophysiology_browser/php/annotations.class.inc

```diff
@@ -27,14 +27,14 @@ class Annotations extends \NDB_Page
      */
     public function handle(ServerRequestInterface $request): ResponseInterface
     {
-        $parameters = $request->getQueryParams();
-        $factory    = \NDB_Factory::singleton();
-        $user       = $factory->user();
-        $db         = $factory->database();
+        $factory = \NDB_Factory::singleton();
+        $user    = $factory->user();
+        $db      = $factory->database();
```

Comment:
Hi @laemtl you could do the following instead of initializing the factory.
```
$user = $request->getAttribute('user');
$db    = $request->getAttribute("loris")->getDatabaseConnection();
```